src/wireviz/wv_helper.py

```diff
@@ -113,6 +114,13 @@ def tuplelist2tsv(inp, header=None):
 def index_if_list(value, index):
     return value[index] if isinstance(value, list) else value
 
```

Comment:
I suggest replacing the second last code line of `tuplelist2tsv()` above with:
```py
output = output + '\t'.join(str(remove_links(item)) for item in row) + '\n'
```
Otherwise, the links will be included in the `.bom.tsv` output file.